systemdspawner/systemdspawner.py

```diff
@@ -211,6 +221,12 @@ def start(self):
                 for path in self.readwrite_paths
             ])
 
+        if self.unit_extra_properties is not None:
+            cmd.extend([
+                self._expand_user_vars('--property={prop}'.format(prop=prop))
```

Comment:
Do we need to do any shell escaping here?

Reply:
Probably. Some of the {prop}s can have spaces in them.
Is there a shlex-method already used by systemdspawner?

Reply:
Nope, but you should be able to import shlex and use shlex.quote.